source/kraken/fill_disruption_from_database.h

```diff
@@ -73,9 +73,8 @@ struct DisruptionDatabaseReader {
 
     std::set<std::string> message_ids;
     std::set<std::tuple<std::string, std::string, std::string>> properties;
-    std::set<std::string> pt_object_ids;
```

Comment:
I guess it was not used

Reply:
This variable was used before, but not anymore.